doc/vundle.txt

```diff
@@ -263,6 +271,12 @@ Installs or updates the configured plugins. Press 'u' after updates complete
 to see the change log of all updated bundles. Press 'l' (lowercase 'L') to
 see the log of commands if any errors occurred.
 
+To update specific plugins, write their names separated by space:
+>
+  :PluginInstall! vim-surround vim-fugitive
```

Comment:
Sometimes, the name_spec can be really long (e.g. When you add repositories from outside GitHub or from local machine).
The name is must be unique and known by the user, so for updating it is a better choice, I think, than specifing the full spec name.